UPGRADE-5.0.md

```diff
@@ -119,6 +119,7 @@ DoctrineBridge
    injected instead
```

Comment:
looks like this item should be on 4.4, and the message should be updated for 5.0, can you do it here too please?
please check the msg just below while doing so. thanks :)

Reply:
I've added deprecation messages to 4.4 and updated them in 5.0. Please check them now.